app/models/event.rb

```diff
@@ -17,4 +18,22 @@ def before_status_full
   def after_status_full
     STATUSES[after_status]
   end
+
+  def new_for?(user)
+    notifications.where('created_at > ? and user_id = ?',
+                        user.last_check_notifications_at, user.id).present?
+  end
+
+  def unread_by?(user)
+    notifications.unread.find_by(user_id: user.id).present?
+  end
+
+  private
+
+  def create_notifications
+    users = issue.followers(User)
+    users.each do |u|
```

Comment:
I thought about it. If we will get done with all features on TODOlist i will improve this. I must use background workers, am I right?